src/components/dataVisualizations/columnHelpers/commanderMatchupsColumnHelper.tsx

```diff
@@ -45,7 +45,7 @@ export const commanderMatchupsColumns: ColumnDef<CommanderMatchupItem, any>[] =
         cell: (info) =>
             info.row.original.matchCount > 0
                 ? `${Math.round((info.row.original.winCount / info.row.original.matchCount) * 100)}%`
-                : `0%`,
+                : `N/A`,
```

Comment:
can we leave a comment here that says this should never happen-- the only things that should be going into this table are commanders with matches against this commander, something weird must have happeend for a commander with no matches to sneak into this table.

something like 

// this should never happen and something went wrong

Reply:
i'm also pretty sure this should never happen considering the match count is already filtered for >0 but there's also a chance that this happens when a player plays invalid matches only. i need to investigate. but should basically never happen.

Reply:
yeah looks like you're correct, it shouldn't happen, ever.